File: sparkiesTraining/trainingCamp/Dia 3/g.py
from sys import stdin


def main():
    n, k = map(int, stdin.readline().strip().split())

    lst = list(map(int, stdin.readline().split()))
    cont = [0 for i in lst]
    used = [False for i in lst]
    flag = True
    i = 0
    ans = 0
    while flag:
        if i == len(lst):
            i = 0
        if lst[i] >= k and cont[i] < 3:
            lst[i] -= k
            cont[i] += 1
        elif (cont[i] == 3 or lst[i] < k) and not used[i]:
            used[i] = True
        else:
            flag = False
        i +=1
    
    print(sum(lst))

main()

File: sparkiesTraining/trainingCamp/Dia 3/g.cpp
#include<bits/stdc++.h>
using namespace std;

void solve(int n, int k){

    vector<int> lst(n);
    for(int i =0; i< n; i++){
        cin >> lst[i];
    }
    bool flag = true;
    int i = 0;
    for (int i = 0; i < n; i++){
        int j = 0;
        while(lst[i] >= k && j < 3){
            lst[i] -= k;
            j++;
        }
    }
    int ans = 0;
    for (int i = 0; i < n; i++){
        ans += lst[i];
    }
    cout << ans << endl;


    
}

int main(){
    int a,b; cin >> a >> b;
    solve(a,b);
        
    return 0;
}